modules/upgrade/pages/migrate-4x-4x.adoc

```diff
@@ -7,8 +7,8 @@ Migrating {productname} from version 4 to 4.x (when available) can be done usin
 
```

Comment:
I am quite confused about this document. Is this for migration the base OS or the product from 4 to 4.x?

I can see we don't talk, for example, about DB update.

Reply:
this is about maintenace window as a concept (buzzword) ... schedule downtime, do the "real things", become available again.

The "real things" are discussed in the linked files such as upgrade:server-update.adoc and upgrade:migrate-4x-4x.adoc.  Of course, everything is connected somehow, but I do not want to discuss everything everywhere ;)

Reply:
No, what I mean is that I don't see anything about the DB update at ` upgrade:migrate-4x-4x.adoc` :-)

Reply:
ah, ok.  will take a closer look and probably reference server-update.adoc.

Reply:
> No, what I mean is that I don't see anything about the DB update at  `upgrade:migrate-4x-4x.adoc` :-)

Because you are asking I assume neither `yast migration` nor `zypper migration` run the schema upgrade?  Please confirm :) @juliogonzalez

Reply:
Schema upgrades are indeed a separate step not triggered by any package, so `zypper` will not perform them.

Reply:
ok, I added some such remark.  I hope it is clear enough now.